tests/BUILD

```diff
@@ -1,6 +1,11 @@
 cc_test(
     name = "simulator-test",
-    srcs = ["simulator_test.cpp"],
+    srcs = [
+        "ccnode_test.cpp",
```

Comment:
nit: typically an individual test rule is created for each test file in the BUILD

Then, if you want to run all the tests in the directory (and sub-directory), you would do `bazel test //tests/...

Reply:
Cool. This is done.